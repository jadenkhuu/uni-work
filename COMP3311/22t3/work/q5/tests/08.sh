python3 ./q5 Randwick 2019-12-26
