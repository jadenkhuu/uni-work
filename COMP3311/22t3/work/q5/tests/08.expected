Race meeting at Randwick on 2019-12-26

Race 1, prize pool $8000, run over 1700m
Neigh Sayer ridden by Josephine Gordon wins $5600
Dâ€™Apples Are Sweet ridden by Anna Lee Aldred wins $1600
Gladiateur ridden by Harry Wragg wins $800

Race 2, prize pool $5000, run over 2200m
Edgar Allen Pony ridden by Blake Shinn wins $3500
Reya Sunshine ridden by Patricia Cooksey wins $1000
Sofa Can Fast ridden by Jimmy Quinn wins $500

Summer Cup, prize pool $160000, run over 2000m
Galtee More ridden by Randy Romero wins $112000
Sir Visto ridden by Mick Dittman wins $32000
Russel ridden by Alex Solis wins $16000
